README.md

```diff
@@ -373,6 +375,9 @@ IEEE Int. Symp. Circuits and Systems (ISCAS), Baltimore, MD, 2017.
 - <a name="Haessig17arxiv"></a>Haessig, G., Cassidy, A. Alvarez, R., Benosman, R., Orchard, G.,  
 *[Spiking Optical Flow for Event-based Sensors Using IBM's TrueNorth Neurosynaptic System](https://arxiv.org/pdf/1710.09820.pdf)*,  
 arXiv:1710.09820, 2017.
+- <a name="Zhu18rss"></a>Zhu, A., Yuan, L., Chaney, K., Daniilidis, K.,  
+*EV-FlowNet: Self-Supervised Optical Flow Estimation for Event-based Cameras*,  
+Robotics, Science and Systems 2018. [PDF](https://arxiv.org/abs/1802.06898), [YouTube](https://youtu.be/eMHZBSoq0sE)
```

Comment:
Thanks. Can you write the same format as our RSS'15 paper? Thanks. In your case:
Robotics: Science and Systems XIV (RSS), Pittsburgh (PA), USA, 2018.

Reply:
Updated.